src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetProcessor.kt

```diff
@@ -43,6 +43,9 @@ class SublimeSnippetProcessor private constructor(sublimeSnippetFile: Path) {
         template.isToIndent = true
         template.isDeactivated = false
 
+        val description = sublimeSnippetDom.getChild("description") ?: null
```

Comment:
FYI, I changed this line to `val description = sublimeSnippetDom.getChild("description")`, without the `?: null` here it also works.